ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v2/SubmitterInfo.scala

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.participant.state.v2
+
+import com.daml.ledger.participant.state.v1.{ApplicationId, CommandId, Offset, Party}
+
+/** Information provided by the submitter of changes submitted to the ledger.
+  *
+  * Note that this is used for party-originating changes only. They are
+  * usually issued via the Ledger API.
+  *
```

Comment:
```suggestion
/** Information provided by the submitter of changes submitted to the ledger.
  * 
  * Note that the submitter is the process that acts as a client of the Ledger API server and not a specific party.
```

Reply:
I have taken that over from the `v0.Update` definition. Should we change it there, too? I'd rather rename the class than to provide this clarification because `actAs`'s documentation also uses the word "submit" and they are Daml parties.